README.md

```diff
@@ -45,6 +45,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <sub><b>Ali Solanki</b></sub>
                 </a>
             </td>
+            <td align="center">
```

Comment:
Please add your name to the end of the list

Reply:
> Please add your name to the end of the list

i had already added my name and github link, in the required area, if it still required, can you please specify where to add